openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -654,3 +848,228 @@ def _execute(
             "generation": self._inputs["generation"],
             **outputs,
         }
+
+
+class GromacsMDRunUnit(gufe.ProtocolUnit):
+    """
+    Protocol unit for running plain MD simulations (NonTransformation)
+    in Gromacs.
+    """
+
+    def _run_gromacs(
+        self,
+        mdp,
+        in_gro,
+        top,
+        tpr,
+        out_gro,
+        xtc,
+        trr,
+        cpt,
+        log,
+        edr,
+        shared_basebath,
+    ):
+        assert os.path.exists(in_gro)
+        assert os.path.exists(top)
+        assert os.path.exists(mdp)
+        p = subprocess.Popen(
+            [
+                "gmx",
```

Comment:
style maybe here

Reply:
Possibly `"-f", mdp` on the same line?

Reply:
`pre-commit.ci` automatically reverts the changes to this unfortunately.